burn-book/src/building-blocks/tensor.md

```diff
@@ -13,14 +13,85 @@ Tensor<B, D, Bool>     // Bool tensor
 Note that the specific element types used for `Float`, `Int`, and `Bool` tensors are defined by
 backend implementations.
 
-## Operations
+In contrast to PyTorch where a Tensor is characterised by the size of it's each dimension, a Burn Tensor is
+characterized by the number of dimensions D in its declaration. The actual shape of the tensor is inferred from
```

Comment:
I don't think PyTorch is characterized by the size of its each dimension. There is nothing specified for the element type, the shape, or the rank. All of those are runtime properties. In contrast, in Burn, we have those specified as types!